app/views/users/index.html.erb

```diff
@@ -1,4 +1,5 @@
 <section id="body-content" class="container">
+  <%= link_to "Create New Post", new_user_post_path(current_user), class: "btn btn-primary" %>
```

Comment:
- [ ] It would be nice to move the button to `create new post` to each user's profile, currently, when I try to create a post I see this error, kindly make sure that a post can be created successfully on each user's profile👍🏿 

![Screenshot_221](https://github.com/AbuTalha3/my_blog_rails/assets/69638013/75c611a6-46b6-4cb7-ad90-dc5413820305)